frontend/src/layouts/authentication/components/LCFSLogin.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import { Link } from 'react-router-dom';
+import { useKeycloak } from '@react-keycloak/web';
+
+// Constants
+import { IDENTITY_PROVIDERS } from 'constants/auth';
+// BC @mui components
+import LoginLayout from 'layouts/authentication/LoginLayout'
+import BCBox from 'components/BCBox';
+import BCButton from 'components/BCButton';
+import BCTypography from 'components/BCTypography';
+
+// @mui material components
+import { Card } from '@mui/material';
+
+// Images
+import logoDark from 'assets/images/gov3_bc_logo.png';
+
+function getSeason(date) {
```

Comment:
something a little more compact (untested):
`
function getSeason(date) {
  const month = date.getMonth() + 1;
  const day = date.getDate();

  return (month === 3 && day >= 20) || (month > 3 && month < 6) || (month === 6 && day <= 20) ? 'spring'
         : (month === 6 && day >= 21) || (month > 6 && month < 9) || (month === 9 && day <= 21) ? 'summer'
         : (month === 9 && day >= 22) || (month > 9 && month < 12) || (month === 12 && day <= 20) ? 'autumn'
         : 'winter';
}
`

Reply:
Thanks, replaced it.